src/navigation/AppNavigator.tsx

```diff
@@ -49,7 +50,7 @@ export default function AppNavigator() {
         {authState.userToken == null ? (
           // TODO: replace with AuthStackNavigator once styled
           // auth screens have functionality integrated
-          <AuthDemo />
+          <AuthStackNavigator />
```

Comment:
<img width="327" alt="image" src="https://user-images.githubusercontent.com/21160510/202799651-c923b2a2-06c8-4743-a85b-1174a1464012.png">


One thing I forgot to add to the `AuthStackNavigator` that is in the other stack navigators (my bad): in order to remove the default navigation header bar (white bar at the top of the screenshot), we have to screenoptions to 
`<AuthStack.Navigator>` in `AuthStackNavigator.tsx`:

```jsx
    <AuthStack.Navigator
      screenOptions={{
        headerShown: false,
      }}
    >
      <AuthStack.Screen 
   ...

```